src/cdk-experimental/package.json

```diff
@@ -19,7 +19,7 @@
     "@angular/cdk": "workspace:*"
   },
   "dependencies": {
-    "tslib": "^2.3.0"
+    "tslib": "2.8.1"
```

Comment:
@devversion we do not want these to be pinned. I think keeping the replace strategy is the way to go, if we want certain deps to be pinned we should do so manually (Like we do in the CLI)